1. .Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16.Найти доверительный интервал для оценки математического ожидания с надежностью 0.95,если выборочная средняя M = 80, а объем выборки n = 256.

In [3]:
import numpy as np

In [24]:
M = 80
stdd = 16
a = 5
a_2 = a/2
z = 1.96
n = 256

In [25]:
M+(z*stdd/np.sqrt(n))

81.96

In [26]:
M-(z*stdd/np.sqrt(n))

78.04

с вероятностью 95% интервал [78.04; 81.96] захватит истинное генеральной совокупности

2. В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью,получены опытные данные:6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей,оценить истинное значение величины X при помощи доверительного интервала, покрывающего этозначение с доверительной вероятностью 0,95.

In [27]:
x = np.array([6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1])

In [28]:
x.sort()

In [29]:
sr1 = np.mean(x)
sr1

6.590000000000001

In [30]:
sr_std1 = np.std(x, ddof=0)
sr_std1

0.4276680956068621

In [31]:
M2 = 6.59
stdd2 = 0.43
n2 = 10

In [32]:
M2+(z*stdd2/np.sqrt(n2))

6.856516761198991

In [33]:
M2-(z*stdd2/np.sqrt(n2))

6.323483238801009

с вероятностью 95% интервал [6.32; 6.86] захватит истинное генеральной совокупности

3. Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм.Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметроказался равным 17.5 мм, а дисперсия известна и равна 4 кв.мм.

H0: диаметр = 17 мм
H1: диаметр = 17.5 мм

In [34]:
0.5/0.1

5.0

In [35]:
Z = 1.654

In [36]:
5>1.654

True

H0 не верна. H1 верна.

4. Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек.
Вес каждой пачки составляет:
202, 203, 199, 197, 195, 201, 200, 204, 194, 190. Известно,
что их веса распределены нормально.
Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

In [37]:
x1 = np.array([202, 203, 199, 197, 195, 201, 200, 204, 194, 190])

In [38]:
x1.sort()

In [39]:
Mx1 = np.mean(x1)
Mx1

198.5

In [43]:
sr_stdx1 = np.std(x1, ddof=0)
sr_stdx1

4.224926034855522

In [47]:
Mx1 = 198.5
stdx1 = 4.2
nx1 = 10

In [40]:
1.5/0.1

15.0

In [41]:
a1 = 1
a1_2 = 0.5
100-0.5

99.5

In [52]:
zx1 = 2.57

In [53]:
Mx1-(zx1*stdx1/np.sqrt(nx1))

195.08663749361426

In [49]:
Mx1+(zx1*stdx1/np.sqrt(nx1))

201.91336250638574

Утверждение продавца верно, так как 200 г попадает в доверительный интервал